data/css/_colors.scss

```diff
@@ -1,3 +1,5 @@
+$brand-color: #005a71 !default;
```

Comment:
What's `$brand-color` meant for? It may be worth adding in a separate commit with explanation about what it should be used for.